### ISSUE TITLE
Training code of baseline model

### ISSUE THREAD

**Issue by kalpeshjp89** [2Obe/BSData#2]
Hello,
I am trying to train the mask-RCNN but because of very small ROI I am facing difficulties to achieve comparable results. It is better if you could share the training code or hyper-parameters of your models to reproduce it.

Thanks in advance.

**Comment by abhiqaxq** [comment]
Hello sir have you completed this ml model
